tests/addresslib/address_test.py

```diff
@@ -138,7 +138,7 @@ def test_edge_cases():
 def test_display_name__to_full_spec():
     eq_('"foo (\\"bar\\") blah" <foo@bar.com>',
         EmailAddress('foo ("bar") blah', 'foo@bar.com').full_spec())
-    eq_('"foo. bar" <foo@bar.com>',
+    eq_('foo. bar <foo@bar.com>',
```

Comment:
Some mailbox providers are being helpful and adding quotes when decoding some base64 encoded display names. This adds a second set of quotes if we already added them and that caused an issue for at least one customer. We cant just not add the quotes in the general case but I think we can safely unquote the strings where the only special character is a `.`.